09Ianuarie/src/ro/sci/ale/ian09/validation/DataValidationTest.java

```diff
@@ -0,0 +1,18 @@
+package ro.sci.ale.ian09.validation;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class DataValidationTest {
+
+    @Test
+    void validateDateTrue() {
+        assertTrue(DataValidation.validateDate(02, 03));
+    }
+
+    @Test
+    void validateDateFalse() {
+        assertFalse(DataValidation.validateDate(-02, -03));
+    }
```

Comment:
I would add a couple of other tests to make sure that all cases are covered, I would test also the following:
16:30 ---- assertTrue
25:33 ---- assertFalse
23:75 ---- assertFalse